crates/uv-interpreter/src/virtual_env.rs

```diff
@@ -20,6 +19,19 @@ pub struct Virtualenv {
 }
 
 impl Virtualenv {
```

Comment:
Is calling it `Virtualenv` still correct?

Reply:
I'm open to other names. What would you suggest? `PythonEnvironment`?

Reply:
`InterpreterRoot` maybe? Almost tempted to name it something related to site packages